modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DocumentLibrary/DocumentLibrary.scss

```diff
@@ -0,0 +1,20 @@
+.liferay-ddm-form-field-document-library {
```

Comment:
@alinedoleron, a CSS class with the same name already exists [here](https://github.com/liferay/liferay-portal/blob/aeadbfb6f2563a86deb75befd0ecd3b1dd2284eb/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/css/main.scss#L220-L224), should we keep both? Can we merge it to a single place? Maybe we can keep this class on the existent **main.scss**.